pt/lessons/advanced/typespec.md

```diff
@@ -74,9 +74,9 @@ defmodule Examples do
 end
 ```
 
-Já definimos o tipo `t(first, last)`, que é uma representação da estrutura `%Examples{first: first, last: last}`. Nesse ponto, vemos tipos que podem receber parâmetros, mas definimos o tipo `t` e nesse momento, ele é uma representação da estrutura `%Examples{first: integer, last: integer}`.   
+Já definimos o tipo `t(first, last)`, que é uma representação da estrutura `%Examples{first: first, last: last}`. Nesse ponto, vemos tipos que podem receber parâmetros, mas definimos o tipo `t` e nesse momento, ele é uma representação da estrutura `%Examples{first: integer, last: integer}`.
 
-Qual a diferença? A primeira representa a estrutura `Examples` e as duas chaves poderiam receber qualquer tipo. A segunda representa a estrutura que as chaves são `integers`. Que significa um código como este:
+Qual a diferença ? A primeira representa a estrutura `Examples` e as duas chaves poderiam receber qualquer tipo. A segunda representa a estrutura que as chaves recebem são do tipo `integers`. Que significa um código como este:
```

Comment:
We don’t need this extra space before the ?

Reply:
indeed I commit this change and remove the blank space, thanks a lot for your review @vbrazo